DOCS/man/options.rst

```diff
@@ -5275,8 +5274,9 @@ them.
         (This filter is an alias for ``jinc``-windowed ``jinc``)
 
     ``ewa_lanczossharp``
-        A slightly sharpened version of ewa_lanczos. If your hardware can run
-        it, this is probably what you should use by default.
+        A slightly sharpened version of ewa_lanczos, preconfigured to use an
+        ideal radius and parameter. This is the default when using
```

Comment:
Note: `ewa_lanczossharp` now matches `ewa_lanczos` radius, which is why I removed the "ideal radius" wording from this paragraph.

Reply:
Oh, looks like rebase issue, I reverted your change.